clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -21,6 +21,22 @@ import shapeless.CNil
   */
 sealed trait ClioCommand
 
+sealed trait GetSchemaCommand extends ClioCommand {
+  def index: TransferIndex
+}
+
+sealed trait AddCommand extends ClioCommand {
+  def index: TransferIndex
+  def key: TransferKey
+  def metadataLocation: String
+}
+
+sealed trait QueryCommand extends ClioCommand {
+  def index: TransferIndex
+  def queryInput: Any
```

Comment:
Could you add a `FIXME` or `TODO` comment here?

Reply:
Yes